devicetypes/smartthings/fortrezz-water-valve.src/fortrezz-water-valve.groovy

```diff
@@ -35,7 +35,7 @@ metadata {
 
 	// tile definitions
 	tiles {
-		standardTile("contact", "device.contact", width: 2, height: 2, canChangeIcon: true) {
+		standardTile("contact", "device.valve", width: 2, height: 2, canChangeIcon: true) {
```

Comment:
do you want to change the tile name to contact and if you want, the type to multiAttributeTile?